### PR TITLE
New APi default list

### PR DESCRIPTION
**Pull request by sumanvpacewisdom**

### CODE REVIEW

File: src/constants/common.js [hunk 1]
```diff
@@ -50,7 +50,7 @@ module.exports = {
 		'/user/v1/account/registrationOtp',
 		'/user/v1/account/resetPassword',
 		'/user/v1/admin/login',
-		'/user/v1/userRole/list',
+		,
 	],
```
Comment (nevil-mathew) [inline src/constants/common.js:54]: rmove this ','

File: src/constants/common.js [hunk 2]
```diff
@@ -63,6 +63,8 @@ module.exports = {
 		'/user/v1/admin/triggerPeriodicViewRefreshInternal',
 		'/user/v1/account/search',
 		'/user/v1/organization/list',
+		'/user/v1/userRole/default',
+		'/user/v1/userRole/list',
```
Comment (nevil-mathew) [inline src/constants/common.js:67]: Are these 2 apis internally accessed

File: src/controllers/v1/userRole.js [hunk 3]
```diff
@@ -85,4 +85,13 @@ module.exports = class userRole {
 			return error
 		}
 	}
+
+	async default(req) {
```
Comment (nevil-mathew) [inline src/controllers/v1/userRole.js:89]: add jsdoc

**Comment by sumanvpacewisdom** [comment]
@nevil-mathew I have updated pr with  testcases , api doc , postman collections

**Comment by nevil-mathew** [comment]
Reviewed and approved

**Comment by sumanvpacewisdom** [comment]
@rakeshSgr Nevil has approved , please take it forward .

**Pull request merged**